packages/mdl-button/mdl-button.scss

```diff
@@ -60,15 +59,7 @@
     opacity: .12;
   }
 
-  &::after {
-    width: 100%;
-    height: 100%;
-    background: black;
-    opacity: 0;
-    overflow: hidden;
-  }
-
-  &:active::after {
+  &:active::before {
     opacity: .06;
```

Comment:
This is actually decreases the opacity, when what we really want is to increase it. I think you mean `.18` here (`.12` + another `.06`)